sdk/eventhub/event-hubs/src/linkEntity.ts

```diff
@@ -142,8 +142,11 @@ export class LinkEntity {
     if (this._context.tokenCredential instanceof SharedKeyCredential) {
       tokenObject = this._context.tokenCredential.getToken(this.audience);
       tokenType = TokenType.CbsTokenTypeSas;
-      // renew sas token in every 45 minutess
-      this._tokenTimeoutInMs = (3600 - 900) * 1000;
+
```

Comment:
I'd recommend a comment here so that a future version of our enthusiastic selves does not try to optimize this and remove it :)

```suggestion
// expiresOnTimestamp can be 0 if the token is not meant to be renewed
```